docs/source/Plugin/P052_S11.rst

```diff
@@ -0,0 +1,141 @@
+.. include:: ../Plugin/_plugin_substitutions_p05x.repl
+.. _P052_S11_page:
+
+S11
+===
+
+|P052_typename|
+|P052_status|
+
+.. image:: P052_S11_1.jpg
+
+Introduction
+------------
+
+This is the flagship CO2 sensor from Swedish company Senseair.
+
+Specifications:
+ * 400 to 5000 ppm
+ * 5V (3.05-5.5VDC)
+ * Peak 125mA, avarage 38μA
+ * 11.5 x 33.5 x 19.7 mm
+ * UART/I2C communication
+ * More than 15 years life span
+ * No burn-in needed, will stabilize after 8 days at worst depending on placement environment
+
+Wiring
+------
+
+Circuit wiring (S11 seen from top, with the 5 vias left and the 4 vias right):
+
+.. code-block:: html
+
+    ESP               S11 (UART)
+    GPIO (14)  <-->   TX (5th via of the 5 vias)
+    GPIO (12)  <-->   RX (4th via [below TX])
+
+    Power
+    5.0V       <-->   VBB (2nd via of the 5 vias, 3.3V can also be used)
+    GND        <-->   GND (1st via)
```

Comment:
Beter name it 'hole' instead of 'via'.
It took me a while to realize what you were trying to say, even though it is technically correct to call it a via.

Reply:
We might need to change this on all the plugin pages... I only changed it for this one... We will need to come up with a terminology look up table for users to look at. But that's for later :)